opentreemap/importer/models/species.py

```diff
@@ -263,6 +263,7 @@ def validate_itree_code_field(self):
         else:
             self.cleaned[fields.species.ITREE_PAIRS] = pairs
 
+    @transaction.atomic
```

Comment:
Interesting, I just assumed that the usual rule applies - use transactions to avoid inconsistent states arising from relationships that are not enforced at the schema level. Is your point that a transaction is not necessary because, while incorrect, it's not the worst thing ever if one succeeds and one fails?

Do you have some reason to believe that just wrapping the saves in a transaction is substantively faster? It comes at the cost of readability. It might be the case that the transaction is initiated at the first db call, which should result in the same performance.